### PR TITLE
Refactor(eos_designs): Make DHCP default route generated by default

### PR DESCRIPTION
**Pull request by gmuloc**
## Change Summary

Change default for `dhcp

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
* change default value in schema

## How to test
molecule

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ wan_router:
             - name: Ethernet1
               wan_carrier: Inmrasat
               wan_circuit_id: S511
-              dhcp_accept_default_route: true
+              dhcp_accept_default_route: false
               ip_address: dhcp
             - name: Ethernet2
               wan_carrier: AWS-1
```
Comment (ayushmittal-arista) [inline ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml:107]: Remove `dhcp_accept_default_route: true` from Ethernet2, to test that default value is respected.
Currently we have no test case for it.

Comment (gmuloc) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml:107]: we do in the other host `node-type-l3-interfaces`

Comment (ayushmittal-arista) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml:107]: Yup I see the updated molecule now.
Thanks

**Review by ayushmittal-arista — APPROVED** [review]

**Review by carl-baillargeon — APPROVED** [review]
LGTM

**Pull request merged**